wandb/lib/globals.py

```diff
@@ -12,3 +13,11 @@ def set_global(run=None, config=None, log=None, join=None, summary=None):
         wandb.join = join
     if summary:
         wandb.summary = summary
+
+
+def unset_globals():
```

Comment:
@raubitsj wanted to flag this.  We may want to do this in `wandb.join` as well, currently only using in tests.